### ISSUE TITLE
NII2BIDS should check for missing NII files before starting

### ISSUE THREAD

**Issue by jan-petr** [ExploreASL/ExploreASL#839]
### Description

* Another issue in NII2BIDS is that a folder (e.g., `perf` or `anat`) is created before checking if the data is complete. These folders should only be created after checking if the respective `temp` data are complete (i.e., just before moving the data from `/temp`).

* Also, this now issues a warning about a missing file (see below) Whereas the whole folder is missing. I would rewrite this warning that this whole scantype was incomplete (in this example, `ASL4D.json` is not the only missing file but the whole ASL DICOM series were missing. It does say this correctly in the dcm2nii conversion part, but helpful to state this correctly here as well).
 
```
Warning: Missing file:
/scratch/hjmutsaerts/EMBARC/Example_TX/temp/TX0061_1/ASL_1/ASL4D.json 
> In xASL_imp_NII2BIDS_RunPerf (line 49)
  In xASL_imp_NII2BIDS_Run (line 37)
  In xASL_imp_NII2BIDS_Session (line 65)
  In xASL_imp_NII2BIDS_Subject (line 48)
  In xASL_imp_NII2BIDS (line 70)
  In xASL_module_Import (line 208)
  In ExploreASL_ImportMaster (line 36)
  In ExploreASL_Master (line 71)
  In ExploreASL (line 14) 
```
* this should then issue a warning about incomplete data in `temp` rather than stating a specific file (often the whole run is missing, not only 1 file).

### Requirements

Optional: add bug requirements here

### How to test

Optional: insert description about how to test the code changes here

### Release notes

Required: summarize the changes for the release notes here
